magic-link/templates/post-connections.php

```diff
@@ -263,32 +263,34 @@ public function body() {
                     <button type="button" class="mdi mdi-information-outline" onclick="document.getElementById('post-detail-modal')._openModal()"></button>
                 </header>
                 <div id="search-filter">
-                    <!--
                     <div id="search-bar">
-                        <input type="text" id="search" placeholder="Search" />
+                    <input type="text" id="search" placeholder="Search" onkeyup="searchChange(<?php echo esc_html( $this->post['ID'] ) ?>)" />
+                        <button class="clear-button mdi mdi-close" onclick="clearSearch()"></button>
```

Comment:
Can we hide this when the input is empty? It can't be cleared when it's already empty, so it doesn't make sense to show it then. Onkeyup, you can make it show if there is a value or hide it if empty. On clear, you can hide it.